terraform/elasticsearch.tf.json.template.py

```diff
@@ -94,8 +94,10 @@
                         "ebs_enabled": "true",
```

Comment:
Do these ebs_options need to be removed off? 

The R6GD instance types require Elasticsearch 7.9 or later or any version of OpenSearch and do not support EBS storage volumes.

https://docs.aws.amazon.com/opensearch-service/latest/developerguide/supported-instance-types.html

Reply:
The conditional below should take care of that.